requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.15.2
+numpy==1.16.4
```

Comment:
AH i see you used requirements.txt locally. environment.yml was on par. These should be consistent, yes.

Reply:
Correct. I checked environment.yml, which seemed to be okay. Didn't checked other dependencies, don't accept just yet. I will check the other libraries (seaborn seems to be missing, for instance)

Reply:
Finalized comparing requirements.txt and environment.yml   ---   Re-requested review